daq/test_modules/native_module.py

```diff
@@ -0,0 +1,45 @@
+"""Module for running docker-container tests"""
+from __future__ import absolute_import
+
+import os
+import logger
+
+from .native_host import make_native_host
+from .external_module import ExternalModule
+
+
+LOGGER = logger.get_logger('docker')
+
+
+class NativeModule(ExternalModule):
+    """Class for running native tests"""
+
+    # pylint: disable=too-many-arguments
+    def __init__(self, host, tmpdir, test_name, module_config, basedir, startup_script):
+        super().__init__(host, tmpdir, test_name, module_config, basedir=basedir)
+        self.startup_script = startup_script
+
+    def start(self, port, params, callback, finish_hook):
+        """Start the native test"""
+        super().start(port, params, callback, finish_hook)
+        LOGGER.debug("activating native test %s", self)
+
+    def _get_env_vars(self, params):
+        env_vars = super()._get_env_vars(params)
+        env_vars.append(('TEST_ROOT', self.basedir))
+        return env_vars
+
+    def _get_vol_maps(self, params):
+        vol_maps = super()._get_vol_maps(params)
+
+        # Common testing tools
+        vol_maps.append((os.path.abspath('bin/retry_cmd'), '/bin/retry_cmd'))
+        vol_maps.append((os.path.abspath('resources/setups/baseline/module_manifest.json'),
```

Comment:
Why not remove all module_manifests and store them directly in module_configs.

Reply:
as mentioned in the other comment, a manifest is conceptually different than config

Reply:
I really wish this could be discussed 5 days ago. Based on the comments, it looked like you were fine with this when I took a lot time to make all the changes. 
Base on the conceptually differences described, the manifest should really be into the new .daqmodule file; however this is also another big change.